cryptofeed/rest/kraken.py

```diff
@@ -0,0 +1,279 @@
+from time import time
+import hashlib, hmac, requests, urllib, json, base64
+
+from cryptofeed.rest.api import API
+from cryptofeed.feeds import KRAKEN
+from cryptofeed.log import get_logger
+
+LOG = get_logger('rest', 'rest.log')
+
+class Kraken(API):
+    ID = KRAKEN
+
+    api = "https://api.kraken.com/0"
+
+    def _post_public(self, command: str, payload = {}):
+        url = "{}{}".format(self.api, command)
+
+        resp = requests.post(url, data = payload)
+
+        if resp.status_code != 200:
+            LOG.error("%s: Status code %d", self.ID, resp.status_code)
+            LOG.error("%s: Headers: %s", self.ID, resp.headers)
+            LOG.error("%s: Resp: %s", self.ID, resp.text)
+            resp.raise_for_status()
+
+        return resp.json()
+
+
+    def _post_private(self, command: str, payload = {}):
```

Comment:
setting default args to empty dicts are bad news. Set them to None and set to {} inside of the func if necessary